### PR TITLE
made the page blue

### PR DESCRIPTION
**Pull request by AnishVV**

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -29,7 +32,7 @@
          }
         </style>
     </head>
-    <body>
+    <body id="page">
```
Comment (gracegao72) [inline templates/index.html:35]: why is this here

Comment (AnishVV) [reply templates/index.html:35]: oh yep you're, this slipped in, let me remove it
